content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+and Composite Resources. Providers are responsible for reconciling their managed
+resources. 
+{{< /hint >}}
+
+Crossplane monitors resources in one of two ways:
+1. A Kubernetes _[watch](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes)_
+2. Periodic polling
+
+Crossplane requests that Kubernetes notifies Crossplane of any changes on
+objects. This notification tool is a _watch_. 
+
+Watched objects include Providers and Composite Resource Definitions.
+
+For objects that Kubernetes can't provide a watch for, Crossplane
+periodically poll the resource to find it's state. The default polling rate is
+one minute. Change the polling rate with the `---poll-interval` pod argument.
+
+Managed resources use polling.
+
+Crossplane checks all resources, both watched and polled resources, to
+confirm they're in the desired state. Crossplane does this every one hour by
+default. Use the `--sync-interval` Crossplane pod argument to change this
+interval. 
+
+The most that Crossplane acts on objects is the 
+`--max-reconcile-rate` rate. This rate is 10 times per second by default.
+
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+## RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+The Crossplane RBAC manager pod automates required Kubernetes RBAC permissions
+for Crossplane and Crossplane Providers. 
+
+{{<hint "note" >}}
+Crossplane installs and enables the RBAC manager by default. You can disable the
+RBAC manager pod.  
+Disabling the RBAC manager requires manual Kubernetes permissions definitions
+for proper Crossplane operations. 
+
+The 
+[RBAC manager design document](https://github.com/crossplane/crossplane/blob/master/design/design-doc-rbac-manager.md) 
+provides more comprehensive details on the Crossplane RBAC requirements.
+{{< /hint >}}
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC init pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager requires the `CompositeResourceDefinition` and
+`ProviderRevision` resources to be available before starting.
+
+The RBAC manager init pod waits for these resources before starting the main 
+RBAC manager pod. 
+
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager pod preforms the following tasks:
+* creating and applying RBAC policies to Provider Service Accounts, allowing 
+  them to control their managed resources
+* allowing the `crossplane` Service Account to create managed resources
+* creating and applying policies for Crossplane Composite Resources
+
+When installing a Provider their service account needs permissions to create and 
+manage managed resources. 
+
+When creating Composite Resources the Crossplane service account needs
```

Comment:
Why do we capitalize Composite Resources here, yet above we write managed resources in lowercase?